pkg/rbac/rbac.go

```diff
@@ -190,18 +190,16 @@ func NewEnforceHandler(basePath string, enforcer *casbin.Enforcer) func(c echo.C
 			return false, errors.New("invalid URL")
 		}
 		switch resource {
-		case "backup-storages", "monitoring-instances":
-			object = "*"
-		case "database-clusters",
-			"database-engines",
-			"database-cluster-backups",
-			"database-cluster-restores":
+		// These resources are not namespaced.
+		case "namespaces",
+			"backup-storages",
+			"monitoring-instances":
```

Comment:
Actually, we discussed that even though the `BackupStorages` and `MonitoringInstances` themselves aren't namespaced we would still enforce some sort of namespacing based on the `.spec.allowedNamespaces`. It doesn't need to be in this PR but we need to think about how to implement this...

Reply:
Created a ticket: https://perconadev.atlassian.net/browse/EVEREST-1250